pkg/v1/providers/ytt/03_customizations/02_avi/add-ako.yaml

```diff
@@ -0,0 +1,21 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:json", "json")
+
+#@ load("ako-deployment.lib.yaml", "ako_namespace", "ako_serviceaccount",
+#@    "ako_configmap", "ako_clusterrole", "ako_clusterrolebinding",
+#@    "ako_statefulset", "ako_ingressclass",
+#@    "hostrules_crd", "httprules_crd", "aviinfrasettings_crd")
+
+#! render the ako artifacts
+#@ if data.values.PROVIDER_TYPE == "vsphere" and data.values.TKG_CLUSTER_ROLE != "workload" and data.values.AVI_ENABLE and data.values.AVI_CONTROL_PLANE_HA_PROVIDER:
+--- #@ ako_namespace()
+--- #@ ako_serviceaccount()
```

Comment:
you can combine service accont, cluster role and cluster role binding together as rbac to reduce function you are using.